Gemfile

```diff
@@ -24,13 +24,22 @@ if File.exist?(file)
 else
   Bundler.ui.warn "[EngineCart] Unable to find test application dependencies in #{file}, using placeholder dependencies"
 
-  gem 'rails', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
+  if ENV['RAILS_VERSION']
```

Comment:
This is from upstream in `engine_cart`. I think the goal is to /not/ generate any of these dependencies unless `ENV['RAILS_VERSION']` is explicitly specified.